packages/bigtable/src/family.js

```diff
@@ -61,27 +62,6 @@ function Family(table, name) {
     create: true,
 
     /**
-     * Delete the column family.
-     *
-     * @param {function=} callback - The callback function.
-     * @param {?error} callback.err - An error returned while making this
-     *     request.
-     * @param {object} callback.apiResponse - The full API response.
-     *
-     * @example
-     * family.delete(function(err, apiResponse) {});
-     */
-    delete: {
```

Comment:
I looked at the new delete method but couldn't figure out why it can't still be done this way.